amundsen_application/base/base_bigquery_preview_client.py

```diff
@@ -0,0 +1,65 @@
+from http import HTTPStatus
```

Comment:
any file needs to add a license header like other file. Just copy paste should be good enough.

Reply:
```
# Copyright Contributors to the Amundsen project.
# SPDX-License-Identifier: Apache-2.0
```

Reply:
I see. thx